modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -927,6 +933,42 @@ public void execute(Project project) {
 
 			});
 
+		String bundleChecksumMD5 = _getBundleChecksumMD5(project);
+
+		download.doLast(
+			new Action<Task>() {
+
+				@Override
+				public void execute(Task task) {
+					if (Objects.isNull(bundleChecksumMD5)) {
+						return;
+					}
+
+					List<File> downloadFileList = download.getOutputFiles();
+
+					if (downloadFileList.isEmpty()) {
+						return;
+					}
+
+					File[] downloadFiles = downloadFileList.toArray(
+						new File[0]);
+
+					verifyBundleTask.src(downloadFiles[0]);
+
+					verifyBundleTask.algorithm("MD5");
```

Comment:
this should be done in a `_setupTaskVerifyBundle`. See other tasks for the pattern.